Sources/Purchasing/StoreKitAbstractions/PromotionalOffer.swift

```diff
@@ -42,15 +43,30 @@ public final class PromotionalOffer: NSObject {
 
 // MARK: - SignedData
 
-extension PromotionalOffer {
+@objc public extension PromotionalOffer {
 
     /// Contains the details of a promotional offer discount that you want to apply to a payment.
-    internal struct SignedData {
-        var identifier: String
-        var keyIdentifier: String
-        var nonce: UUID
-        var signature: String
-        var timestamp: Int
+    @objc(RCPromotionalOfferSignedData)
+    class SignedData: NSObject {
```

Comment:
I had made this `internal` on purpose to show that it was just an implementation detail, but I guess if the hybrids need it...